src/base/rss/rss_article.cpp

```diff
@@ -104,6 +106,10 @@ bool Article::isRead() const
     return m_isRead;
 }
 
+QString Article::value(const QString key) const {
+    return m_varHash.value(key, QString('_')).toString();
```

Comment:
Just an empty string would cause the download to fail (converting '/folder/<placeholder>/whatever' to '/folder//whatever', for example). I wasn't sure how it would handle a blank space (' '). But I figured an underscore is easy enough to see. It can also just be the value of the "key" parameter.